docs/morsePersistence.md

```diff
@@ -1,63 +1,57 @@
 
 The simplified persistence diagram is then used as a constraint for the [TopologicalSimplification](https://topology-tool-kit.github.io/doc/html/classttkTopologicalSimplification.html) of the input scalar data.
 
-This simplified data is then used as the input of the computation of [ScalarFieldCriticalPoints](https://topology-tool-kit.github.io/doc/html/classttkScalarFieldCriticalPoints.html) (top left view, above screenshot) and the [ContourTree (FTM)](https://topology-tool-kit.github.io/doc/html/classttkFTMTree.html) (bottom left view, above screenshot).
+This simplified data is then used as the input of the computation of [MorseSmaleComplex](https://topology-tool-kit.github.io/doc/html/classttk_1_1MorseSmaleComplex.html) (centrale view, above screenshot). This complex is composed of elements of 3 dimensions: dimension 0, which corresponds to the critical points of the Morse-Smale Complex, dimension 1, which corresponds to its edges (in grey on the screenshot) and dimension 2, which corresponds to its surfaces.
 
 ## ParaView
 To reproduce the above screenshot, go to your `ttk-data`  directory and enter the following command:
 ``` bash
-$ paraview states/dragon.pvsm
+$ paraview states/morsePersistence.pvsm
 ```
 
 ## Python code
 
 ``` python  linenums="1"
---8<-- "python/dragon.py"
+--8<-- "python/morsePersistence.py"
 ```
 
 ## Inputs
-- [dragon.vtu](https://github.com/topology-tool-kit/ttk-data/raw/dev/dragon.vtu): a two-dimensional triangulation.
+- None
 
 ## Outputs
 - `PersistenceDiagram.vtu`: the output persistence diagram in VTK file format (bottom right view, above screenshot). You are free to change the `vtu` file extension to that of any other supported file format (e.g. `csv`) in the above python script.
 - `PersistenceCurve.csv`: the output persistence curve.
-- `CriticalPoints.vtp`: the output
-critical points
-in VTK file format (bottom right view, above screenshot). You are free to change the `vtp` file extension to that of any other supported file format (e.g. `csv`) in the above python script.
-- `ContourTreeNode.vtp`: spheres, representing the nodes of the output contour tree
-in VTK file format (bottom right view, above screenshot). You are free to change the `vtp` file extension to that of any other supported file format (e.g. `csv`) in the above python script.
-- `ContourTreeArcs.vtp`: cylinders, representing the arcs of the output contour tree
-in VTK file format (bottom right view, above screenshot). You are free to change the `vtp` file extension to that of any other supported file format (e.g. `csv`) in the above python script.
-
+- `MorseComplexeCriticalPoints.vtp`: the output critical points (or 0 dimensional elements) of the Morse Smale Complex in VTK file format (centrale view, above screenshot). You are free to change the `vtp` file extension to that of any other supported file format (e.g. `csv`) in the above python script.
```

Comment:
centrale -› center (?)

Reply:
see my remarks in your scripts about the graphical primitives (not necessary in batch mode).
in batch mode, the raw output data (here points) is a more generic and useful representation.